docs/samples/Microsoft.ML.Samples/Microsoft.ML.Samples.csproj

```diff
@@ -3,6 +3,7 @@
   <PropertyGroup>
     <TargetFramework>netcoreapp2.1</TargetFramework>
     <OutputType>Exe</OutputType>
+    <WarningsNotAsErrors>649</WarningsNotAsErrors>
```

Comment:
><WarningsNotAsErrors>649</WarningsNotAsErrors> [](start = 4, length = 46)

If you're getting errors because of the data classes you create, you can solve this by doing a `public xyz Xyz {get; set;}` in your classes.

Reply:
Thanks Rogan. Leave this here, so people don't keep bumping into this? 

---
In reply to: [267927422](https://github.com/dotnet/machinelearning/pull/3042#discussion_r267927422) [](ancestors = 267927422)